src/clib/lib/enkf/gen_obs.cpp

```diff
@@ -102,7 +100,7 @@ static void gen_obs_set_data(gen_obs_type *gen_obs, int buffer_size,
 }
 
 void gen_obs_load_observation(gen_obs_type *gen_obs, const char *obs_file) {
-    auto vec = gen_common_fload_alloc(obs_file, gen_obs->obs_format);
+    auto vec = gen_common_fload_alloc(obs_file);
     gen_obs_set_data(gen_obs, vec.size(), vec.data());
```

Comment:
This will segfault when buffer_size is odd. We should create an issue for it. Also, buffer can very easily be converted to a `vector`.

Reply:
https://github.com/equinor/ert/issues/5424